lib/ngx/balancer_stream.lua

```diff
@@ -0,0 +1,82 @@
+-- Copyright (C) Yichun Zhang (agentzh)
```

Comment:
Yes, thats my intention. Once you, I or someone else implements the subsystem check.